src/main/java/study/step3/domain/Car.java

```diff
@@ -0,0 +1,17 @@
+package study.step3.domain;
+
+public class Car {
+
+
+
+    public int getPosition() {
+        return position;
+    }
+
+    private int position = 1;
+
+    public void move(){
+        this.position += 1;
+    }
```

Comment:
기본 컨벤션에 따르면 클래스 변수, 멤버변수, 생성자, 메서드 순으로 배치하고 있어요
```suggestion
    private int position = 1;

    public int getPosition() {
        return position;
    }
    
    public void move(){
        this.position += 1;
    }
```